frontend/src/Comment.js

```diff
@@ -0,0 +1,35 @@
+import './comment.css';
+
+function Comment({ children }) {
```

Comment:
The name of your component is very general. Are you sure it is handling "comments" of any types?

Reply:
Done